src/ResourceManager/MachineLearningCompute/Commands.MachineLearningCompute/packages.config

```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.MachineLearningCompute" version="0.1.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.MachineLearningCompute" version="0.2.0" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.10" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net452" />
```

Comment:
@shutchings same comment about removing the added references

Reply:
Got it.